docs/en-US/component/tree.md

```diff
@@ -35,6 +35,14 @@ tree/custom-leaf
 
 :::
 
+## Lazy loading multiple times ^(2.6.2)
```

Comment:
@tolking  The version number has not been updated, maybe it needs to be updated. 🤣